app/javascript/dashboard/routes/dashboard/captain/responses/Index.vue

```diff
@@ -174,15 +182,28 @@ const handleCardSelect = id => {
   bulkSelectedIds.value = selected;
 };
 
+const fetchResponseAfterBulkAction = () => {
+  // Only fetch if no records left
+  if (filteredResponses.value?.length === 0) {
+    const page =
+      responseMeta.value?.page > 1
+        ? responseMeta.value.page - 1
+        : responseMeta.value.page;
+    fetchResponses(page);
+  }
+
+  // Clear selection
+  bulkSelectedIds.value = new Set();
+};
```

Comment:
I think in general we should refetch, say if there are only 2 items left, or just 1, it shouldn't matter. Let's refetch after the bulk actions anyway